docs/entities/tokens/non-fungible-token-history-event-0-tx-id.schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "NonFungibleTokenHistoryEventWithTxId",
+  "description": "Non-Fungible Token history event with transaction id",
+  "required": ["event_index", "asset_event_type_id", "tx_id"],
+  "additionalProperties": false,
+  "properties": {
+    "sender": {
+      "type": "string"
```

Comment:
What do you think about adding `"nullable": true` for fields like this (like when a token is minted and has no sender)?

It's not technically part of OpenAPI spec, but I think redoc and some other tooling supports it.

Reply:
That would be cool, I'll look into it